<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="group1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>541</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushStat">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : white; border : 2px solid black; color : black;}
QWidget:hover{background-color : rgb(200, 200, 200);}
QWidget:pressed{background-color:rgb(50, 50, 50);}</string>
    </property>
    <property name="text">
     <string>주문 통계</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : white; border : 2px solid green; color : green;}
QWidget:hover{background-color: rgb(151, 206, 132);}
QWidget:pressed{background-color : rgb(0, 100, 0);}</string>
    </property>
    <property name="text">
     <string>가동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSearch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : white; border : 2px solid black; color : black;}
QWidget:hover{background-color : rgb(200, 200, 200);}
QWidget:pressed{background-color:rgb(50, 50, 50);}</string>
    </property>
    <property name="text">
     <string>물품 찾기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : red; border : 2px solid red; color : white;}
QWidget:hover{background-color: rgb(240, 141, 141);}
QWidget:pressed{background-color: rgb(164, 7, 7);}</string>
    </property>
    <property name="text">
     <string>긴급 정지</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : green;</string>
    </property>
    <property name="text">
     <string>가동중</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOff">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red;</string>
    </property>
    <property name="text">
     <string>대기중</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Vemana2000</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 18, 106);</string>
    </property>
    <property name="text">
     <string>가산</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../data/resources.qrc">:/icon/mail.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Vemana2000</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 18, 106);</string>
    </property>
    <property name="text">
     <string>HUB</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 30px; background-color : white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Hub 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 30px; background-color : white; </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Hub 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 30px; background-color : white; </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Hub 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub22">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelhub33">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>296</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>분류 중 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>296</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>541</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushSearch2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius :35px; border : 1px solid black; background-color : white; }
QWidget:hover{background-color : rgb(200, 200, 200);}
QWidget:pressed{background-color:rgb(50, 50, 50);}</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>481</width>
      <height>161</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>hub</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>state</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>treatment</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>start_time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>end_time</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="cbSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbHub">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="editStart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>3</month>
      <day>7</day>
     </date>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="editEnd">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
      <year>2024</year>
      <month>3</month>
      <day>13</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>3</month>
      <day>13</day>
     </date>
    </property>
    <property name="time">
     <time>
      <hour>23</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStop2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : red; border : 2px solid red; color : white;}
QWidget:hover{background-color: rgb(240, 141, 141);}
QWidget:pressed{background-color: rgb(164, 7, 7);}</string>
    </property>
    <property name="text">
     <string>긴급 정지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStart2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : white; border : 2px solid green; color : green;}
QWidget:hover{background-color: rgb(151, 206, 132);}
QWidget:pressed{background-color : rgb(0, 100, 0);}</string>
    </property>
    <property name="text">
     <string>가동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMain">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메인화면</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOff2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red;</string>
    </property>
    <property name="text">
     <string>대기중</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOn2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : green;</string>
    </property>
    <property name="text">
     <string>가동중</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>크기 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>허브 : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbState">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCategory">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>상태 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>취급 :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbChecktime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushReset">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>374</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>521</width>
     <height>811</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushStart3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : white; border : 2px solid green; color : green;}
QWidget:hover{background-color: rgb(151, 206, 132);}
QWidget:pressed{background-color : rgb(0, 100, 0);}</string>
    </property>
    <property name="text">
     <string>가동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStop3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{border-radius : 15px; background-color : red; border : 2px solid red; color : white;}
QWidget:hover{background-color: rgb(240, 141, 141);}
QWidget:pressed{background-color: rgb(164, 7, 7);}</string>
    </property>
    <property name="text">
     <string>긴급 정지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMain2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>710</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메인화면</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbList">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Graphlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>481</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelOn3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : green;</string>
    </property>
    <property name="text">
     <string>가동중</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOff3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : red;</string>
    </property>
    <property name="text">
     <string>대기중</string>
    </property>
   </widget>
   <widget class="QLabel" name="label8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>통계 기준 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>처리일 : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDay">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../data/resources.qrc"/>
 </resources>
 <connections/>
</ui>